proposals/0216-qualified-do.rst

```diff
@@ -316,7 +316,7 @@ raise an error *even if there is a value for ``(>>=)`` in scope*.
 
 Enabling ``-XQualifiedDo`` doesn't change the meaning of existing do-expressions.
 
-When both ``-XQualifiedDo`` and ``-XRebindableSyntax`` are enabled, ``-XQualifiedDo`` only affects qualified ``do``'s and ``-XRebindableSyntax`` affects the unqualified ``do``'s.
+When both ``-XQualifiedDo`` and ``-XRebindableSyntax`` are enabled, ``-XQualifiedDo`` only affects qualified ``do``s and ``-XRebindableSyntax`` affects the unqualified ``do``s.
```

Comment:
Not sure if this renders right, do we need a zero-width space here?

Reply:
An escaped space `\ ` works.

Reply:
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#escaping-mechanism